src/Simplex/Chat/Messages.hs

```diff
@@ -719,6 +719,12 @@ data ChatPagination
   | CPBefore ChatItemId Int
   deriving (Show)
 
+data ChatPaginationTs
+  = CPLastTs Int
+  | CPAfterTs ChatItemTs Int
+  | CPBeforeTs ChatItemTs Int
```

Comment:
it may be not chat list specific but then it could be called Pagination instead of ChatPagination, and chat pagination already exists and is done for items inside chat by id, not timestamp, so it's somewhat misleading. Actually even synonym ChatItemTs is incorrect here semantically because chat list is paginated by chat timestamp, not chat item timestamp - it could be simply UTCTime.